f5lbaasdriver/test/tempest/services/clients/bigip_client.py

```diff
@@ -196,3 +196,34 @@ def virtual_server_has_policy(self, vs_name, policy_name, partition):
                     return True
 
         return False
+
+    def virtual_server_has_profile(self, vs_name, profile_name, partition):
+        if self.virtual_server_exists(name=vs_name, partition=partition):
+            vs = self.bigip.tm.ltm.virtuals.virtual.load(
+                name=vs_name, partition=partition)
+
+            profiles = vs.profiles_s.get_collection()
+            for profile in profiles:
+                if profile.name == profile_name:
+                    return True
+
+        return False
+
+    def virtual_server_has_persist(self, vs_name, persist, partition):
```

Comment:
Apparently not. It was added to support testing persistence profiles, which is missing from the test. I will update the test.